Makefile.org

```diff
@@ -424,6 +424,14 @@ clean:	libclean
 	rm -fr $$i/*; \
 	done
 
+distclean: clean
+	-$(RM) `find . -type l -a \! -path "./.git/*"`
```

Comment:
Trouble with -path is that it appears to be GNU-ism, i.e. it's not guaranteed to work universally...